src/internal/components/ThemeEditor.tsx

```diff
@@ -74,10 +74,7 @@ export const ThemeEditor = (props: ThemeEditorProps) => {
       const localHistories = JSON.parse(localHistoryArray) as History[];
       const localHistoryIndex = localHistories.length - 1;
       if (localHistoryIndex >= 0) {
-        devLogger.logData(
-          "LOCAL_VISUAL_CONFIGURATION_DATA",
-          localHistories[localHistoryIndex].state
-        );
+        devLogger.log("Theme Dev Mode - Using layout data from local storage");
```

Comment:
We still want to log the data though. Are we doing that?

Reply:
Whenever we receive themeData or visualConfigurationData it gets logged by the receiver.

We don't log in this specific case where we use the localStorage though, but there is a message to let the user know that it is using localStorage and they can access the message in console log if they wish.